test/f32-vmax.cc

```diff
@@ -1316,6 +1316,183 @@
 #endif  // XNN_ARCH_WASM || XNN_ARCH_WASMSIMD || XNN_ARCH_WASMRELAXEDSIMD
 
 
+TEST(F32_VMAX__HVX_U32, batch_eq_32) {
```

Comment:
This is failing internal presubmits because this needs to be wrapped in an `#if XNN_ARCH_HEXAGON` or similar, because it's trying to run these tests on inappropriate architectures.

Reply:
(Same with the other tests too)

Reply:
Thank you Dillon! I just fixed the error and generated the proper if macro for HVX kernels.

Reply:
Clearly the new change did not fix the error. Do you see any new errors from build bot result? I tried to build locally for X86 and checked if any build/ctest fails for f32-vbinary but seems all fine.

Reply:
Actually it looks like you did fix the errors.

This failure is different, and unrelated to your change. You don't need to make any changes to fix it. The system tries to ignore pre-existing failures like this when testing new changes, but it's not always able to do that. I'll check with this change in a bit and see if I can make it retry the internal tests.